packages/asc-ui/src/components/ErrorMessage/ErrorMessage.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import ErrorMessage from './ErrorMessage'
+
+export default {
+  title: 'Experimental/Atoms/ErrorMessage',
+  decorators: [
+    (storyFn: () => React.ReactNode) => (
+      <div
+        style={{
+          display: 'flex',
+          maxWidth: 400,
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          padding: '40px 10px',
+        }}
+      >
+        {storyFn()}
+      </div>
+    ),
+  ],
+}
+
+export const Default = () => <ErrorMessage message="Dit is dus fout" />
+
+export const NoMessage = () => <ErrorMessage />
```

Comment:
Hi @jasperswart, can you add stories to show how it looks when used with for example `Checkbox`, `Radio`, and/or `TextField` components?

Reply:
I have added the examples for input, select, radio and checkbox to story

Reply:
can you check?

Reply:
I like it very much